### PR TITLE
Feat/st 553 changed text on preview draft order

### PR DESCRIPTION
**Pull request by muradalamv1**
Enter a description.
- Changed draft order preview text
- order tab has been changed 

(https://tools.hmcts.net/jira/browse/ST-553)

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 
 
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/CriminalInjuriesCompensation.java:19]: Revert this change? It's just whitespace

File: src/main/java/uk/gov/hmcts/sptribs/document/content/DocmosisTemplateProvider.java [hunk 2]
```diff
@@ -13,5 +13,6 @@ public class DocmosisTemplateProvider {
 
     public String templateNameFor(final String templateId, final LanguagePreference languagePreference) {
         return docmosisTemplatesConfig.getTemplates().get(languagePreference).get(templateId);
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/document/content/DocmosisTemplateProvider.java:16]: Just a whitespace change? Rollback?

File: src/main/java/uk/gov/hmcts/sptribs/document/content/DraftEditTemplateContentCIC.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
+//package uk.gov.hmcts.sptribs.document.content;
+//
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/document/content/DraftEditTemplateContentCIC.java:2]: Why is everything commented out?

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Why the completely commented out class?

File: src/main/java/uk/gov/hmcts/sptribs/document/content/DraftEditTemplateContentCIC.java (hunk 3 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/document/content/DraftEditTemplateContentCIC.java:2]: this is very old code coming from master.I updated from master and now my branch has very old code .Dnt know why
